docs/src/administration/cli/api-tokens.md

```diff
@@ -154,18 +167,10 @@ To run a cron only on your production environment, check the environment type as
 ```yaml
 crons:
     backup:
-        spec: 'H 4 * * *'
+        spec: '0 0 * * *'
         commands:
             start: |
                 if [ "$PLATFORM_ENVIRONMENT_TYPE" = production ]; then
-                   platform backup:create --yes --no-wait
+                   platform source-operation:run update --no-wait --yes
```

Comment:
Probably not the best example. Maybe you have a better one to suggest @CollierCZ?

Reply:
Just realized that it's [not recommended to run source operations on production environments](https://docs.platform.sh/create-apps/source-operations.html#automated-source-operations-using-cron). So I changed the example, hopefully it is suitable in this case (my knowledge of cron jobs is very limited).